usr/bin/tailscale

```diff
@@ -31,7 +31,7 @@ if [ ! -f /tmp/tailscale ]; then
         echo -e "tailscale_${version}/tailscaled" >> /tmp/tailscale_${version}_files.txt
     fi
 
-    wget -O- https://pkgs.tailscale.com/stable/tailscale_${version}.tgz | tar x -zvf - -C /tmp -T /tmp/tailscale_${version}_files.txt
+    wget -O- https://dl.tailscale.com/stable/tailscale_${version}.tgz | tar x -zvf - -C /tmp -T /tmp/tailscale_${version}_files.txt
```

Comment:
Where do you see this URL? I still see `pkgs` on their site: https://pkgs.tailscale.com/stable/#static

Reply:
Looks like your URL redirects to pkgs with `?cdn=0`, not sure what this fixes for you, could you elaborate?